lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -101,6 +100,8 @@ def trigger
       puppet_trigger['start_date'] = self.class.normalized_date("#{trigger['start_year']}-#{trigger['start_month']}-#{trigger['start_day']}")
       puppet_trigger['start_time'] = self.class.normalized_time("#{trigger['start_hour']}:#{trigger['start_minute']}")
       puppet_trigger['enabled']    = trigger['flags'] & Win32::TaskScheduler::TASK_TRIGGER_FLAG_DISABLED == 0
+      puppet_trigger['minutes_interval'] = trigger['minutes_interval'] ||= 0
+      puppet_trigger['minutes_duration'] = trigger['minutes_duration'] ||= 0
```

Comment:
You're right that `0` and `1440` are not interchangeable, and the tasks are different.

## GoogleUpdateTaskMachineUA

![screen shot 2015-04-02 at 9 30 55 am](https://cloud.githubusercontent.com/assets/394081/6968362/15d78666-d91b-11e4-882a-1b1e5037e63c.png)


## GoogleUpdateTaskMachineCore

![screen shot 2015-04-02 at 9 32 27 am](https://cloud.githubusercontent.com/assets/394081/6968380/408da98a-d91b-11e4-92f5-a04e12d66ccf.png)